pkg/watchdog/watchdog-report.sh

```diff
@@ -63,13 +67,27 @@ if [ -n "$bootReason" ]; then
     # Do not overwrite an existing file since it is likely to be more
     # specific like Fatal
     if [ -f /persist/boot-reason ]; then
-        echo "Watchdog report not replacing $(cat /persist/boot-reason) with $bootReason" >>/persist/log/watchdog.log
+        log "Watchdog report not replacing $(cat /persist/boot-reason) with $bootReason"
     else
         echo $bootReason > /persist/boot-reason
-        echo "Watchdog report saved $bootReason" >>/persist/log/watchdog.log
+        log "Watchdog report saved $bootReason"
     fi
 fi
 
+log "Run collect-info.sh now to capture the state before reboot"
+exec_id="$(basename "$(mktemp)")"
+if output=$(
+    ctr --namespace services.linuxkit t exec --exec-id "$exec_id" debug \
+        /containers/services/debug/rootfs/usr/bin/collect-info.sh -t 0 2>&1
+); then
+    log "collect-info.sh succeeded (exec-id=$exec_id). Output:"
+    log "$output"
```

Comment:
you don't need to repeat `log "$output"` if you are always calling it.... just move it to outside the `if`...